deps/rabbitmq_cli/lib/rabbitmq/cli/core/command_modules.ex

```diff
@@ -74,8 +74,31 @@ defmodule RabbitMQ.CLI.Core.CommandModules do
           []
       end
 
+    all_plugins = PluginsHelpers.list(opts)
```

Comment:
This change means we either have to always have the plugins directory set, including in all kinds of test environments, or avoid returning an error when the directory does not exist.

Reply:
c2f87ad2d7cec6db2718cc8bcd99177a885ff85d and 0c821ded1a1b4a45d1f98387a4abd15db2757e39 are two different solutions. We are discussing with @pjk25 if we should apply both or just one of them.